tools/check_format.py

```diff
@@ -103,14 +102,12 @@ def checkTools():
                             "users".format(CLANG_FORMAT_PATH))
   else:
     error_messages.append(
-      "Command {} not found. If you have clang-format in version 7.x.x "
-      "installed, but the binary name is different or it's not available in "
-      "PATH, please use CLANG_FORMAT environment variable to specify the path. "
-      "Examples:\n"
-      "    export CLANG_FORMAT=clang-format-7.0.0\n"
-      "    export CLANG_FORMAT=/opt/bin/clang-format-7".format(
-        CLANG_FORMAT_PATH)
-    )
+        "Command {} not found. If you have clang-format in version 7.x.x "
```

Comment:
this still looks 4-char indented though maybe that's fine: nested-scope indent=2, continuation-indent=4 sgtm.